app/models/concerns/digital_object_concerns/attribute_assignment/pending_publish_entries.rb

```diff
@@ -22,8 +22,6 @@ def assign_pending_publish_entries(digital_object_data)
         # Raise error if pub_to and unpub_from have any overlapping values
         raise ArgumentError, 'Cannot include the same publish target in publish_to and unpublish_from' if (pub_to & unpub_from).length.positive?
 
-        pub_to.freeze
-        unpub_from.freeze
```

Comment:
what's going on here? do these lists need to be modifiable after creation?

Reply:
I previously made them frozen, but since they're being assigned to `pending_publish_to` and `pending_unpublish_from` and it seemed useful to modify those elsewhere, I undid that.  I think my original freeze choice came from an earlier publication design.